java/core/src/test/java/sleeper/core/record/process/status/TestProcessStatusUpdateRecords.java

```diff
@@ -18,11 +18,13 @@
 
 import java.time.Instant;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+/**
+ * A test helper for creating lists of process status update records.
```

Comment:
It might be helpful to explain this will simulate the contents of a status store.